src/main/webapp/WEB-INF/context/applicationContext.xml

```diff
@@ -65,7 +65,7 @@
         </property>
     </bean>
 
-    <bean id="propertyResolver" factory-bean="primaryPropertyPlaceholderConfigurer" factory-method="getPropertyResolver"/>
+    <bean id="propertyResolver" factory-bean="propertyConfigurer" factory-method="getPropertyResolver"/>
```

Comment:
This change looks familiar -- was this a mixup in the earlier PR adding uP5 properties support?

Reply:
Yes, from another PR. I should split this and some version upgrades into another PR since they are corrections not associated with this work.

Reply:
For my part, I'm not worried about splitting it.  But a note would help me understand the purpose.

Reply:
Quick note on this one: it was a cut and paste where the factory bean differed between this portlet and the origin of the code.